.github/workflows/main.yml

```diff
@@ -9,6 +9,6 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: Rubocop checks
-      uses: mikker/rubocop-action@master
+      uses: mikker/rubocop-action@mikker-patch-1
```

Comment:
what does this mean?

Reply:
Means what git ref to use. I was testing the Rubocop GH action. Removed these commits again.